xls/examples/bitonic_sort.x

```diff
@@ -0,0 +1,220 @@
+////////////////////////////////////////////////////////////////////////////////
+// Bitonic sort
```

Comment:
should we point to https://en.wikipedia.org/wiki/Bitonic_sorter#Example_code as a reference for the algorithm?

Reply:
Done.